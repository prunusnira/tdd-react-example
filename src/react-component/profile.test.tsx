import Profile from './profile';
import {render} from '@testing-library/react'

describe('Profile', () => {
    it('match snapshot', () => {
        const util = render(<Profile username='me' name='test' />);
        expect(util.container).toMatchSnapshot();
    });
    it('check props', () => {
        const util = render(<Profile username='me' name='test' />);
        util.getByText('me');
        util.getByText('(test)');
        util.getByText(/te/);
    })
})