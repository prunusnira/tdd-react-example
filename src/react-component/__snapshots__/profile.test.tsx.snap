// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Profile match snapshot 1`] = `
<div>
  <div>
    <b>
      me
    </b>
    Â 
    <span>
      (
      test
      )
    </span>
  </div>
</div>
`;
